crates/proof-of-sql/src/proof_primitive/dory/setup.rs

```diff
@@ -85,7 +85,7 @@ impl<'a> From<&'a PublicParameters> for ProverSetup<'a> {
             value.Gamma_2_fin,
             value.max_nu,
             #[cfg(feature = "blitzar")]
-            &value.blitzar_handle,
+            value.get_blitzar_handle(),
```

Comment:
Well, I guess since this is needed, the handle can't be truly hidden from `super`.